src/CMakeLists.txt

```diff
@@ -552,7 +552,7 @@ endif(${WITH_LTTNG})
 add_subdirectory(global)
 
 # RGW also support Lua scripting
-if(WITH_CEPHFS OR WITH_RADOSGW)
+if((WITH_CEPHFS OR WITH_RADOSGW) AND WITH_RADOSGW_LUA)
```

Comment:
i think the parens are wrong here. if cephfs requires lua, that shouldn't depend on WITH_RADOSGW_LUA:
```cmake
if(WITH_CEPHFS OR (WITH_RADOSGW AND WITH_RADOSGW_LUA))
```